Makefile

```diff
@@ -30,7 +30,7 @@ image-publish: image
 
 test-e2e:
 	docker-compose up -d
-	@sleep 5
+	@sleep 15
```

Comment:
Let's add something like `wait_for_es_ready` function with a timeout that will monitor the ES until it's ready.